File: demo/src/java/ast/KoKoId.java
/**
 @author loriacarlos@gmail.com
 
*/
package kokoslan.ast;


public class KoKoId extends KoKoAtom<String>{
   public KoKoId(String value){
      super(value);
   }
   @Override 
   public KoKoValue eval(KoKoContext ctx){
	   return ctx.find(this);
   }
   
}

File: KoKoslan/src/kotlin/ast/KoKoId.kt
/** 
   Andrey Arguedas Espinoza
   Daniela Armas Sanchez
   Michael Chen Wang
   Kimberly Olivas Delgado
*/

package kokoslan.kt.ast
import kokoslan.kt.eval.*
import kokoslan.kt.exception.*

class KoKoId(value : String) : KoKoAtom<String>(value) {

   override fun eval(ctx : KoKoContext) : KoKoValue {
	   return ctx.find(this)
   }
   
}